sdks/go/pkg/beam/transforms/sql/sql_test.go

```diff
@@ -0,0 +1,103 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sql
+
+import (
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/transforms/sql/sqlx"
+	"reflect"
+	"testing"
+)
+
+func TestOptions_Add(t *testing.T) {
+	str := "this is a string"
+	bytes := []byte{1, 2, 3, 4}
+	test := struct {
+		opt sqlx.Option
+	}{
+
+		opt: sqlx.Option{
+			Urn:     str,
+			Payload: bytes,
+		},
+	}
+
```

Comment:
I don't mind the indirection for a single test structure, but there's no need to put the values in separate variables first. Imagine there were additional test cases, defining everything with str1, bytes3, ahead of time adds no value and hurts readability.

```suggestion
	test := struct {
		opt sqlx.Option
	}{
		opt: sqlx.Option{
			Urn:     "this is a string",
			Payload: []byte{1, 2, 3, 4},
		},
	}
```
Similarly below in the other tests.